docs/entities/post-conditions/post-condition-2-non-fungible.schema.json

```diff
@@ -4,26 +4,29 @@
   "type": "object",
   "allOf": [
     {
-      "$ref": "./abstract-post-condition.schema.json"
+      "type": "object",
+      "required": ["principal"],
+      "properties": {
+        "principal": {
+          "$ref": "./post-condition-principal.schema.json"
+        }
+      }
     },
     {
       "type": "object",
       "required": ["conditionCode", "amount", "assetInfoId", "assetValue", "asset"],
       "properties": {
         "conditionCode": {
-          "enum": [10, 11],
+          "enum": [16, 17],
```

Comment:
I think since we're expanding the opaque encoding to a readable JSON encoding, we should use descriptive string constants for these. 
`["sent", "not_sent"]` as opposed to numbers. Thoughts?